flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/PredefinedOptions.java

```diff
@@ -221,6 +229,9 @@ public ColumnFamilyOptions createColumnOptions(Collection<AutoCloseable> handles
 
     // ------------------------------------------------------------------------
 
+    public static final long DEFAULT_LOG_FILE_SIZE = 10 * 1024 * 1024L;
+    public static final int DEFAULT_LOG_FILE_NUM = 10;
```

Comment:
I think 10 files with each 10MB at most should be fine in production environment, @NicoK what do you think?